apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerCommandTests.cs

```diff
@@ -1352,7 +1352,9 @@ public async Task CanExecuteReadCommand()
             spannerClientMock.Received().StreamingRead(
                 Arg.Is<ReadRequest>(request => request.Columns.Equals(new RepeatedField<string> { "Col1", "Col2" })), Arg.Any<CallSettings>());
             spannerClientMock.Received().StreamingRead(
-                Arg.Is<ReadRequest>(request => request.KeySet.Equals(new V1.KeySet { Keys = {  new ListValue { Values =
+                Arg.Is<ReadRequest>(request => request.KeySet.Equals(new V1.KeySet
+                {
+                    Keys = {  new ListValue { Values =
```

Comment:
I used the VScode format and using the recommended extension for `editorconfig` and this got changed but I don't think this is correct?